static/src/inline-svgs/illustration/contributions-finance.svg

```diff
@@ -0,0 +1 @@
62.178.513.28.766.197.5.463.982.596 1.5.14.54.124 1.145.858.875.053 1.075-.018 2.105.18 3.084.196.98.66 1.908 1.015 2.89.117-.11.268-.247.442-.41l.53.662h-.558c-.174.936.198 1.675.55 2.52.526 1.26 1.062 2.582 1.08 4.026.006.452.085 1.214.856 1.088.424-.07.954-.36 1.157-.713.48-.833.827-1.75 1.156-2.66.248-.69.348-1.436.584-2.134.386-1.127.802-2.248 1.254-3.352.483-1.188.458-2.554 1.29-3.622.103-.13-.02-.428.08-.558.15-.192.405-.303.73-.525.463.897.675 2.218 1.885 2.65-.16.286-.406.53-.41.782-.02 1.065.403 1.96 1.145 2.74.318.33.52.77.747 1.177.285.5.55 1.013.798 1.53.202.414.558.862.52 1.263-.123 1.16.762 1.757 1.3 2.504.3.414 1.062.464 1.52.805.733.548 1.302.25 1.705-.32.555-.792 1.025-1.66 1.435-2.54.22-.47.09-1.14.375-1.54.66-.93.8-1.97.98-3.034.1-.57.368-1.11.544-1.667.3-.965.59-1.932.896-2.942.484-1.04 1-2.16 1.57-3.396.296-.048.513-.002.46.62-.042.533.176 1.09.307 1.63.064.278.243.535.275.813.117 1.05-.006 2.144.852 3 .2.2.167.613.328.876.584.96-.208 2.558 1.5 3.05.298.088.355 1.003.533 1.534.07.21.167.41.234.617.15.452.494.965.39 1.35-.182.652.125.784.57.967.14.06.25.202.373.305-.397.505-.183 1.016.868 2.16 1.816-.44 2.23-.968 2.325-2.91.01-.215.076-.467.205-.632.46-.584.422-1.337.754-1.995.63-1.26.887-2.705 1.325-4.064.15-.473.604-.976.513-1.373-.23-1.006.562-1.06 1.144-1.587.347 1.262.66 2.39.966 3.522.246.91.396 1.857.744 2.723.344.84.9 1.587 1.332 2.392.198.37.448.777.452 1.167.006 1.43.462 2.64 1.447 3.674.728.767 1.65.795 2.27-.08.42-.595.68-1.308.952-1.994.322-.816.585-1.655.865-2.488.31-.928.616-1.857.913-2.79.023-.08-.05-.195-.08-.293-.244-.78-.107-.98.675-1.017.134-.554.29-1.103.396-1.66.064-.347.052-.706.087-1.06.025-.244.03-.502.116-.724.067-.174.32-.28.372-.454.457-1.505.87-3.026 1.34-4.528.195-.624.496-1.213.804-1.957.322.727.61 1.184.727 1.682.215.912 1.127 1.63.742 2.72-.06.163.185.432.29.65.162.344.333.682.482 1.028.134.295.24.604.367.9.07.15.275.377.242.416-.656.762.22 1.068.456 1.54.426.846 1.3 1.437 1.33 2.522.007.312.346.61.51.925.343.672.928 1.338.958 2.027.087 1.868 1.132 2.862 2.837 3.694.357-.713.833-1.344.99-2.045.357-1.644.566-3.322.832-4.986.1-.625.193-1.25.296-1.874.005-.02.053-.036.06-.058.098-.44.195-.882.287-1.324.045-.222.055-.457.12-.672.35-1.135.82-2.247 1.04-3.405.17-.882.033-1.82.033-2.83.194.134.3.165.323.227.645 1.69 1.29 3.383 1.923 5.08.363.983.63 2.008 1.077 2.95.503 1.062 1.186 2.04 1.747 3.08.628 1.16 1.46 2.293 1.728 3.54.198.916.446 1.214 1.304 1.216.296-.577.6-1.16.956-1.853-.017-.14-.12-.45-.08-.742.15-1.044.356-2.08.522-3.123.03-.183-.05-.38-.043-.57.013-.557-.106-1.188.113-1.655.27-.577.454-1.005-.05-1.42.27-.22.54-.44.804-.66l-.254-.022-.476-1.733.148-.067c.12.11.243.22.41.374.03-.385.038-.69.08-.99.137-1.03.277-2.062.43-3.092.163-1.08.275-2.17.524-3.23.437-1.85 1.3-3.582 1.162-5.587-.07-1.044.303-2.12.493-3.18.163-.885.37-1.763.53-2.65.067-.385.08-.782.116-1.173.6 1.537.987 3.04 1.375 4.547.013.053 0 .112.008.168.12.938.14 1.9.378 2.812.407 1.567.848 3.126 1.293 4.685v-8.738z"/></g></svg>
```

Comment:
I think that if you can't reduce the points in sketch then it makes sense to use PNGs instead if they're that much smaller, what do you reckon?